package-lock.json

```diff
@@ -3170,9 +3347,9 @@
       }
     },
     "node_modules/@uniswap/sdk-core": {
```

Comment:
sor already has transitive dependency on sdk-core through multiple packages, v2-sdk, v3-sdk, etc. All follows the semantic versioning with the caret (^) prefix. I'm wondering if you only change the sdk-core version in `package-lock.json`, would it work?